### ISSUE TITLE
We need nicer bullet point lists

### ISSUE THREAD

**Issue by WolfgangDpunkt** [Zettlr/Zettlr#4205]
### Description

I know, this has already been discussed in places.
But one of the few points that really bother me a lot about the Markdown representation in Zettlr (although I love the app) is the not so nice representation of (bulleted) list.
Due to the fact that they bullets are not UL elements and the list items can also not be controlled via custom.css, because the corresponding selectors are missing, these lists, which are essential for a note-taking program, really don't look readable. 
Everything clumps together, there is neither left nor top/bottom nice spacing. See the example below how the same markdown text is displayed in Zettlr and here in github. The second one is much nicer, or? :-)
And real bullet symbols would be fancy, too, of course.

So if anyone could remedy this, for example by giving the list items class names, that would be super great!

Zettlr:
<img width="650" alt="grafik" src="https://user-images.githubusercontent.com/35533844/225745105-61f39ac8-6b05-415d-bf9e-d5e6b1bc4dae.png">

Github:
Weltliche Herraschft wird passiv ertragen, Christen sollte, ginge es streng nach der Lehre, apolitisch sein.
* Eigentlich gibt es laut Autor [@nitschkeMittelalterChristlich2018] keine wirkliche „politische Theorie“ von christlichen Autoren des Mittelalters, da sie nicht ihr Anliegen war. 
* Außer der Anerkennung dass es *Herrschaft* gibt, weil Gottes Wille ist, dass sie existiert, gibt es laut Autor nicht viel Theorie zur Welt der Politik in den kirchlichen Texten. Das Soziale des Menschen wird über die biblische Charakterisierung erklärt.
 
Das hermeneutische Problem des christlichen Denkens über Politik besteht im Mittelalter darin, dass sich im Kontext der überaus erfolgreichen.

### Proposed Changes

- Take typographical design of lists from Github (or also e.g. Joplin). 
- Or at least allow custom.css access, through unique class names for all visual list components.

### Caveats

It might interfere with existing layouts, I don't know for sure.

### Do you Wish to Attempt Implementing this Yourself?

No

### Zettlr Version

Beta (if applicable)

### Your Platform

- [ ] Windows
- [X] macOS
- [ ] Linux

### Operating System Version

10.15.7

**Comment by halloichbingunnar** [comment]
Yes, this would be a very important enhancement. Bullet point lists with more than two different levels are very hard to read atm because of the alignment…  see example:
<img width="662" alt="Bildschirm­foto 2023-03-28 um 10 22 24" src="https://user-images.githubusercontent.com/64837252/228175221-1ea72cdb-5987-4f61-b0bf-f1864dbe7494.png">

**Comment by WolfgangDpunkt** [comment]
After a very long search I have now found a CSS selector that can address the pseudo list elements.
It's only a quick and dirty solution, but finally I can display my bulleted lists in zettlr as such, which improves the readability enormously.

```
/* nicer vertical and horizontal spacing */

.cm-line:has(.cm-list) {
   margin: 1.3rem 0rem !important;
   padding-left: 2rem !important;

}

/* real bullets */
.cm-list.cm-code-mark,
.cm-list .cm-code-mark,
.cm-list .cm-content-span.cm-code-mark {
   visibility: hidden !important;
}

.cm-line:has(.cm-list):before {
   content: "•";
   visibility: visible;
   color: black;
   font-size: 1.4rem;
   margin-left: -8px;
}
```

<img width="236" alt="grafik" src="https://github.com/Zettlr/Zettlr/assets/35533844/d5b06f34-2192-4de7-b39c-31607352c40d">

**Comment by nathanlesage** [comment]
This has been fixed sometime in the past two years (don't remember exactly which version).
